server/channels/app/post.go

```diff
@@ -2242,12 +2251,16 @@ func (a *App) GetPostInfo(c request.CTX, postID string) (*model.PostInfo, *model
 		HasJoinedChannel:   channelMemberErr == nil,
 	}
 	if team != nil {
-		_, teamMemberErr := a.GetTeamMember(team.Id, userID)
+		teamMember, teamMemberErr := a.GetTeamMember(team.Id, userID)
 
+		teamType := model.TeamInvite
+		if team.AllowOpenInvite {
+			teamType = model.TeamOpen
+		}
```

Comment:
What's the difference between `team.Type` being `model.TeamOpen` and `team.AllowOpenInvite`?

Reply:
I think that the team type is not used really in the product. The only thing that is really check is `AllowOpenInvite` that is what actually means "every user in the system can join the team".

Reply:
Huh, that's interesting

Reply:
We really need to remove team type at some point. I don't think it's possible to even set through the UI, and I don't think we've ever had a clear definition of how it should work